src/main/java/com/suse/saltstack/netapi/datatypes/Job.java

```diff
@@ -1,17 +1,68 @@
 package com.suse.saltstack.netapi.datatypes;
 
+import com.google.gson.annotations.SerializedName;
 import java.util.List;
 
+// todo - handle dates too (they are represented in really
```

Comment:
Haha i know what you mean :) You could use intellijs live templates for that. I'm testing intellij myself right now but i know that eclipse does not recognize it as a todo. 
A relevant discussion [here](https://github.com/SUSE/saltstack-netapi-client-java/pull/44#discussion_r25929839) about todos. The TL;DR its preferred to open an issue to track todos and if we put todos in the code they should reference the issue as shown by @isbm in the discussion.